sui/deploy-contract.js

```diff
@@ -38,27 +38,21 @@ const {
 const PACKAGE_DIRS = ['gas_service', 'test', 'axelar_gateway', 'operators'];
 
 /**
- * Post-Deployment Functions Mapping
- *
- * This object maps each package name to a post-deployment function.
- */
-const POST_DEPLOY_FUNCTIONS = {
-    GasService: postDeployGasService,
-    Test: postDeployTest,
-    Operators: postDeployOperators,
-    AxelarGateway: postDeployAxelarGateway,
-};
-
-/**
- * Command Options Mapping
- *
- * This object maps each package name to a function that returns an array of command options.
+ * Package Mapping Object for Command Options and Post-Deployment Functions
  */
-const CMD_OPTIONS = {
-    AxelarGateway: () => [...DEPLOY_CMD_OPTIONS, ...GATEWAY_CMD_OPTIONS],
-    GasService: () => DEPLOY_CMD_OPTIONS,
-    Test: () => DEPLOY_CMD_OPTIONS,
-    Operators: () => DEPLOY_CMD_OPTIONS,
+const PACKAGE_MAPPING = {
+    CMD_OPTIONS: {
+        AxelarGateway: () => [...DEPLOY_CMD_OPTIONS, ...GATEWAY_CMD_OPTIONS],
+        GasService: () => DEPLOY_CMD_OPTIONS,
+        Test: () => DEPLOY_CMD_OPTIONS,
+        Operators: () => DEPLOY_CMD_OPTIONS,
+    },
+    POST_DEPLOY_FUNCTIONS: {
+        AxelarGateway: postDeployAxelarGateway,
+        GasService: postDeployGasService,
+        Test: postDeployTest,
+        Operators: postDeployOperators,
+    },
```

Comment:
keys should be camel case. I meant,
```
{
  AxelarGateway: {
    cmdOptions: GATEWAY_CMD_OPTIONS,
    postDeployFunction: postDeployAxelarGateway,
  }
}
```
Also, `DEPLOY_CMD_OPTIONS` are common for every contract, so no need to specify here